docker/theia-app-theia-ext/README.md

```diff
@@ -0,0 +1,48 @@
+# Example docker image for theia-trace-extension
+
+This folder contains an example showing how to create a docker image, that bundles at its core a Theia-based application that uses the _Theia Trace Viewer_ front-end.
+
+Notes:
+
+- the image will contain exclusively the trace viewer front-end.
+  If you want to run a complete _trave viewer_ application, you will need a service running the trace-server (not included here);
+
+- the image will use a pre-built, specific version of the _theia-traceviewer_ extension [npm package]
+
+## How to build and run
+
+Build the image and name it _tate_ (short for "theia application - theia extension"). You may use a different name if you prefer.
+
+Try adding `--network host` in case of build failures related to debian packages retrieval:
+
+```bash
+docker build -t tate .
```

Comment:
This doesn't work anymore. The build has to be done from the parent directory using the yarn command.

Reply:
This trick is still valid, but unfortunately did not help in that specific case. The build instructions have been migrated to the "docker" folder README and adjusted, which I think should help avoid this issue.